### DISCUSSION TITLE
Zeplin에서 이미지를 다운받을 때 이미지 확장자 선택 기준은?

### DISCUSSION THREAD

**Discussion by hy57in** [teamsindy20/sobok#38] [Design]
![image](https://user-images.githubusercontent.com/60775453/131290069-adca5a17-10da-403b-8b30-77a609cf3c87.png)

- png : 최적화된 비트맵 이미지
- webp : 비트맵 이미지
- svg : 최적화된 벡터 이미지

**Comment by gwak2837** [comment]
### 벡터 이미지

- SVG

### 비트맵 이미지

- PNG
- WEBP

### 선택 기준

벡터 이미지는 확대 축소 시 이미지를 깨지지 않고 유지하기 위해 사용하고, 비트맵 이미지는 두 확장자 중에 최대 화질의 이미지 파일 크기가 작은 것으로 사용하면 될 거 같아요.
